src/components/SelectionList/types.ts

```diff
@@ -297,6 +297,9 @@ type BaseSelectionListProps<TItem extends ListItem> = Partial<ChildrenProps> & {
 
     /** Ref for textInput */
     textInputRef?: MutableRefObject<TextInput | null>;
+
+    /** Whether to hide the list on initial render */
```

Comment:
Please update this comment to explain a little bit more _why_ someone using the prop would choose true or false.

Reply:
Updated. Please check.